### ISSUE TITLE
Add support for EKS

### ISSUE THREAD

**Issue by eschizoid** [lightbend/cloudflow#50]
The cloudflow ```sbt``` plugin only deploys to GCP. However, it will be ideal to have support for other cloud providers. EKS seems to be mature enough and could be an initial candidate.

**Comment by agemooij** [comment]
Hi @eschizoid . EKS support is mostly a matter of updating the install scripts. The `sbt` plugin just creates images and pushes them to a configurable Docker registry. The `kubectl-cloudflow` plugin can deploy Cloudflow apps to any cluster running the Cloudflow operator. So all we are missing for EKS support is an installation script that supports it.

I am not aware of any major differences when it comes to EKS and GKE so very likely the install script only needs some minor adjustments to be compatible with EKS. Perhaps you could try to install cloudflow on EKS and let us know where it breaks?

**Comment by eschizoid** [comment]
hey @agemooij I would more than happy to work on this. Before that I have a couple of questions for you :)

>So all we are missing for EKS support is an installation script that supports it.
By that, you mean a script that provisions an initila EKS cluster similar to this script https://github.com/lightbend/cloudflow/blob/master/installer/create-cluster-gke.sh?

>I am not aware of any major differences when it comes to EKS and GKE so very likely the install script only needs some minor adjustments to be compatible with EKS.
Unfortunately, EKS does not use ```kubectl``` instead they have their own binary that behind the scenes I believe invokes ```kubectl``` commands. I don't think it will be challenging though.

**Comment by eschizoid** [comment]
@agemooij let me know what you think about #65

**Comment by eschizoid** [comment]
also after deploying the application `sensor-data-scala` it seems that i need kafka up and running:

```
kubectl get pods -n sensor-data-scala              ✘ 130 feature/k8s-support ✚ ✱ ◼
NAME                                                  READY   STATUS             RESTARTS   AGE
sensor-data-scala-file-ingress-bb4f6f96-bt5xj         0/1     Pending            0          13m
sensor-data-scala-http-ingress-557ddb966f-spd8s       1/1     Running            0          13m
sensor-data-scala-invalid-logger-5b8f8b5b67-4r74f     0/1     CrashLoopBackOff   7          13m
sensor-data-scala-merge-749b695f59-c2br8              1/1     Running            0          13m
sensor-data-scala-metrics-55f697f64c-8sxst            1/1     Running            0          13m
sensor-data-scala-rotor-avg-logger-58d6ff8455-zsks5   0/1     Pending            0          13m
sensor-data-scala-rotorizer-6f84fb4479-ptv2r          1/1     Running            0          13m
sensor-data-scala-valid-logger-78d4d94875-7k4cf       0/1     Pending            0          13m
sensor-data-scala-validation-6b8cf4fbd7-ffhsw         1/1     Running            0          13m
```

**Comment by eschizoid** [comment]
Closed by #65
